cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/CfArgumentsCreator.java

```diff
@@ -67,17 +78,27 @@
 
         log.verbose("User has not passed values for arguments ", missingOptions, ", using default values");
 
-        return extendCommandLine(missingOptions, new LinkedList<>(Arrays.asList(args)));
+        // get missing values from config-properties File and extend to get command
+        if (Arrays.asList(args).contains("get")) {
+
+            return extendForGetCommand(missingOptions, new LinkedList<>(Arrays.asList(args)));
+        }
+        // get missing values from the given YAML File and extend to diff/apply command
+        else {
+
+            return extendForDiffAndApplyCommand(missingOptions, new LinkedList<>(Arrays.asList(args)));
+        }
```

Comment:
To improve readability of the comments:
```suggestion
        if (Arrays.asList(args).contains("get")) {
            // get missing values from config-properties File and extend to get command
            return extendForGetCommand(missingOptions, new LinkedList<>(Arrays.asList(args)));
        } else {
            // get missing values from the given YAML File and extend to diff/apply command
            return extendForDiffAndApplyCommand(missingOptions, new LinkedList<>(Arrays.asList(args)));
        }
```

Reply:
You can use import static so that you will not need: 

Arrays.asList(args)) 

instead: 

asList(args)) 
would be enough.

Reply:
done